gh-pages/_sass/layout.scss

```diff
@@ -38,6 +39,12 @@ $site-nav-width: 300px;
 		width: 82px;
 	}
 
+	&::shadow {
+		.inner {
+			padding: 0 $page-padding;
```

Comment:
This is technically redundant but I'd like this value to change for the header if it ever changes for the body, so this should future proof that.